### ISSUE TITLE
Migrate to google-api-ruby-client gem v0.11.0

### ISSUE THREAD

**Issue by kawamoto** [gimite/google-drive-ruby#253]
there is some breaking changes
https://github.com/google/google-api-ruby-client/blob/master/CHANGELOG.md

when I try to create session with credentials (GoogleDrive::Session.from_credentials), it raises an error. backtrace is like below. it's caused by timeout options is moved to `ClientOptions` from `RequestOptions`

```
NoMethodError: undefined method `timeout_sec=' for #<Google::Apis::RequestOptions:0x007f8177b8e8e0>
Did you mean?  timeout
from **/google_drive-2.1.2/lib/google_drive/api_client_fetcher.rb:25:in `initialize'
```

**Comment by akash-cp** [comment]
I am also facing the same issue.

**Comment by jlfenaux** [comment]
Same problem.

**Comment by gimite** [comment]
Fixed in google_drive 2.1.3.
